lib/simple-seeder.js

```diff
@@ -156,12 +160,15 @@ module.exports = class SimpleSeeder {
         }
       }
 
-      for await (const e of snap.createReadStream()) {
+      for await (const e of snap.createReadStream(contentEncoding)) {
         if (e.value.type === 'list') continue // List of lists is not supported
 
         await this.put(e.key, { ...e.value, external: false })
       }
 
+      const allowedPeers = await snap.metadata.get(ALLOWED_PEERS)
```

Comment:
Doesn't need the metadata encoding?

Reply:
Maybe we're missing a snapshot method in seedbee but just keeping it simple for now